README.md

```diff
@@ -43,15 +43,15 @@ brew install cmake ninja
 brew cask install java # required for Bazel
 ```
 
-Additionally, [Bazel](https://www.bazel.build) v0.22.0 is required to build with TensorFlow support. Instructions to download Bazel directly can be found [below](###bazel). You can find instructions for installing CMake, and Ninja directly [below](#build-dependencies) as well.
+Additionally, a [Bazel](https://www.bazel.build) between v0.24.1 and v0.25.2 (inclusive) is required to build with TensorFlow support. Instructions to download Bazel directly can be found [below](###bazel). You can find instructions for installing CMake, and Ninja directly [below](#build-dependencies) as well.
```

Comment:
What does `###bazel` link to?  The link didn't work for me.

Reply:
Should be fixed now! It was trying to use an anchor link to link to in a lower section of the README but the way you do it is `[here](#section)` even if the section is a Header 3 (aka three pound signs)